x/mint/simulation/genesis_test.go

```diff
@@ -40,14 +42,59 @@ func TestRandomizedGenState(t *testing.T) {
 	var mintGenesis types.GenesisState
 	simState.Cdc.MustUnmarshalJSON(simState.GenState[types.ModuleName], &mintGenesis)
 
-	require.Equal(t, "stake", mintGenesis.Params.MintDenom)
-	require.Equal(t, "0stake", mintGenesis.Minter.EpochProvision(mintGenesis.Params).String())
-	require.Equal(t, "0.000000000000000000", mintGenesis.Minter.NextEpochProvisions(mintGenesis.Params).String())
-	require.Equal(t, "0.000000000000000000", mintGenesis.Minter.EpochProvisions.String())
+	const (
+		expectedEpochProvisionsStr      = "7913048388940673156"
```

Comment:
Are these random arbitrary values?

Reply:
This is what gets generated with a seed of `5`. Similar to how it is in the SDK